torch/csrc/distributed/rpc/tensorpipe_utils.cpp

```diff
@@ -230,7 +236,15 @@ std::pair<tensorpipe::Allocation, TensorpipeReadBuffers> tensorpipeAllocate(
 
 Message tensorpipeDeserialize(
     tensorpipe::Descriptor&& tpDescriptor,
-    TensorpipeReadBuffers&& buffers) {
+    TensorpipeReadBuffers&& buffers,
+    const std::shared_ptr<LazyStreamContext>& ctx) {
+
+  // I believe we don't need this today, as unlike pickling, unpickling does
+  // not insert CUDA ops. But still putting this guard here for safety and
+  // potential future changes.
```

Comment:
I'm a bit conflicted about adding this "just in case". I understand the reasoning, but it gives the impression we don't know what our code does and we're shooting in the dark.